iOS/SampleApp/SampleApp/ViewController.swift

```diff
@@ -43,4 +43,17 @@ class ViewController: UIViewController {
             }
         }
     }
+    
+    @IBAction func runFVM(_ sender: UIButton) {
```

Comment:
ok, if it works. The next step is to move it to factory method in the framework as described in [BUG-486908]